### DISCUSSION TITLE
Pairing RF remote through web interface

### DISCUSSION THREAD

**Discussion by morfi11** [arendst/Tasmota#17315] [Q&A] [answered]
Some of my sonoff switches are hard to reach and I would like to add RF remotes without physically accessing the switch. 
Is there, or could there be a way to put the device in RF pairing mode through its web interface?

**Comment by sfromis** [comment, marked as answer]
Questions should not be opened as "issues", as they are meant for having reason to think that there is a bug, but instead in the Discussions part

Anyway, RF pairing for Sonoff switches is happening in an independent chip, which Tasmota has no way of controlling. Thus no way without physical access (unless possibly by modifying the switch hardware).
